docs/decisions/0007-adjustments.rst

```diff
@@ -0,0 +1,39 @@
+0007 Ledger Adjustments
+#######################
+
+Status
+******
+**Accepted** (October 2023)
+
+Context
+*******
+We'd like to have the abilitiy to manually adjust the balance of ledgers
+in a somewhat unstructured way - that is, in a way that doesn't reverse
+an existing transaction.
+
+Decision
+********
+We'll introduce an ``Adjustment`` model:
+
+* This is somewhat different from our existing pattern of modeling any modification
+  to a ledger via models that inherit from ``BaseTransaction`` (like we
+  do with ``Reversal``).
+* Creating an ``Adjustment`` will cause a new ``Transaction`` to be written
+  to the adjustment's ledger object, with the same quantity as the adjustment record.
+  This transaction record is the thing that fundamentally changes the
+  balance of the related ledger. The transaction will be referred to from a foreign key of the ``Adjustment``
+  record.
+* An adjustment must define some enumerated ``reason`` for being created.
+* An ``Adjustment`` *may* refer to a ``transaction_of_interest`` - this is a foreign
+  key to another transaction of note that is relevant to the reason for the
+  creation of the adjustment.  It is not required.
+* An adjustment *may* include some free-text ``notes`` that help to further
+  explain why the adjustment exists.
+
+Consequences
+************
+* Adjustments work in kind of the opposite way of reversals: instead of using an
+  existing ledger-transaction to instantiate a reversal, we'll have a situation
+  where the action of creating an adjustment happens, which has a side-effect
+  of creating a transaction to adjust the ledger balance.
+* Care must be taken to ensure that adjustments are not over-used.
```

Comment:
[suggestion] Is it worth expanding on any risks if/when adjustments are over-used?